CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # React Native Module Changelog
 
+## Version 8.1.0 - August 17, 2020
+Minor release to add a `removeAllListeners(eventType: EventType)` method, fix a problem with the Android React Native location module and to get the following SDK updates:
```

Comment:
we may not want to be too explicit about typescript in this, since not everyone's using it. maybe:

Minor release adding a `removeAllListeners` method, fixing a bug preventing the the location module from loading on android, and bundling the following SDK updates: